api/v1alpha1/bundle_webhook.go

```diff
@@ -76,6 +76,14 @@ func checkBundleSource(r *Bundle) error {
 		if r.Spec.Source.Git == nil {
 			return fmt.Errorf("bundle.spec.source.git must be set for source type \"git\"")
 		}
+		if r.Spec.Source.Git.Directory != "" {
+			if r.Spec.Source.Git.Directory[0] == '/' {
+				return fmt.Errorf("bundle.spec.source.git.directory can be \"/\"")
+			}
+			if len(r.Spec.Source.Git.Directory) > 2 && r.Spec.Source.Git.Directory[:3] == "../" {
+				return fmt.Errorf("bundle.spec.source.git.directory can not start with \"../\"")
```

Comment:
Nit: add a bit more context about why "../" is not allowed.

Also a tip that backticks can be used to define string literals to avoid needing to escape quotation marks within the string.
```suggestion
				return fmt.Errorf(`bundle.spec.source.git.directory begins with "../": directory must define path within the repository`)
```

Reply:
I take your suggestion.  Thanks!